node/src/GlideClient.ts

```diff
@@ -522,8 +524,10 @@ export class GlideClient extends BaseClient {
      * @remarks Since Valkey version 7.0.0.
      *
      * @param libraryCode - The source code that implements the library.
-     * @param replace - Whether the given library should overwrite a library with the same name if it
+     * @param replace - (Optional) Whether the given library should overwrite a library with the same name if it
      *     already exists.
+     * @param decoder - (Optional) {@link Decoder} type which defines how to handle the response.
+     *     If not set, the {@link BaseClientConfiguration.defaultDecoder|default decoder} will be used.
```

Comment:
```suggestion
     * @param options - (Optional) Additional parameters:
     * - (Optional) `replace `: whether the given library should overwrite a library with the same name if it
     *     already exists.
     * - (Optional) `decoder`: see {@link DecoderOption}.
```
looking forward, we need to groom docs satisfy the doc gen - only actual argument names could be referred with `@param`

Reply:
Please apply this format everywhere you have anonymous option combined with `DecoderOption`